zeebe/protocol-impl/src/test/java/io/camunda/zeebe/protocol/impl/JsonSerializableToJsonTest.java

```diff
@@ -2116,6 +2116,8 @@ final var record =
         """
         {
           "partitionId": 1,
+          "queueId": null,
+          "queueInsertionKey": -1,
```

Comment:
❌ This test verifies the mapping of all properties of the `CommandDistributionRecord` is working as expected. We should make sure we set a value for the `queueId` and `queueInsertionKey`